### PR TITLE
Update Home Page UI

### LINKED ISSUE #52: Home Page UI Update
## Issue Type:

- [x] Feature Request

## **Describe the bug**

New card design for all projects with the preview images.

## **Possible solution**

Add one more key in json object for preview images and fetch them in card layout.

## **Screenshots**

- If applicable, add screenshots to help explain your problem.

### PR DESCRIPTION
**Pull request by jigar-sable**
# Description

Card design update with preview image of each templates. Added hover effects.

Fixes #52 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK

### CODE REVIEW

**Comment by jigar-sable** [comment]
@arpit456jain Not added Calculator previews as it will be on different page and will have another script code for fetching.

**Comment by arpit456jain** [comment]
> @arpit456jain Not added Calculator previews as it will be on different page and will have another script code for fetching.

Ok no problem @jigar-sable  so would you like to work on that in a separate issue??

**Comment by ankitapuri** [comment]
> @arpit456jain Not added Calculator previews as it will be on different page and will have another script code for fetching.
@jigar-sable then you will redirect the calcutor to another page after clicking on view ?

**Comment by jigar-sable** [comment]
@ankitapuri all calculators will not be there on the home page, it will have navbar. Calculators are displayed on another page which will have separate JS & JSON file. Someone is working on that issue #46

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (arpit456jain) [inline .vscode/settings.json:3]: @jigar-sable please delete this file

**Review by arpit456jain — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (jigar-sable) [reply .vscode/settings.json]: Done.

**Review by arpit456jain — APPROVED** [review]

**Review by ankitapuri — APPROVED** [review]

**Comment by ankitapuri** [comment]
@arpit456jain u can merge

**Pull request merged**